src/core/importType.js

```diff
@@ -13,8 +13,9 @@ export function isAbsolute(name) {
 }
 
 export function isBuiltIn(name, settings) {
+  const baseModule = name.split('/')[0]
```

Comment:
How would you feel about replacing the base module name logic with the following?

```js
function baseModule(name) {
  const parts = name.split('/')
  if (name.startsWith('@')) { return parts.slice(0, 2).join('/') }
  return parts[0];
}
baseModule('foo')              //=> "foo"
baseModule('foo/bar/baz')      //=> "foo"
baseModule('@bat/foo/bar/baz') //=> "@bat/foo"
```